<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>RaptorLogin</class>
    <widget class="QDialog" name="RaptorLogin">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>312</width>
                <height>312</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>312</width>
                <height>312</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>312</width>
                <height>312</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QHBoxLayout" name="_LoginLayout">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="QWidget" name="_Stage" native="true">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                    <property name="minimumSize">
                        <size>
                            <width>312</width>
                            <height>312</height>
                        </size>
                    </property>
                    <property name="maximumSize">
                        <size>
                            <width>312</width>
                            <height>312</height>
                        </size>
                    </property>
                    <property name="styleSheet">
                        <string notr="true"/>
                    </property>
                    <layout class="QVBoxLayout" name="_StageLayout">
                        <property name="spacing">
                            <number>0</number>
                        </property>
                        <property name="leftMargin">
                            <number>0</number>
                        </property>
                        <property name="topMargin">
                            <number>0</number>
                        </property>
                        <property name="rightMargin">
                            <number>0</number>
                        </property>
                        <property name="bottomMargin">
                            <number>0</number>
                        </property>
                        <item>
                            <widget class="QWidget" name="_Header" native="true">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>50</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>50</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <layout class="QHBoxLayout" name="_HeaderLayout" stretch="8,1,1">
                                    <property name="spacing">
                                        <number>12</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>12</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>12</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QLabel" name="_Title">
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="_HeaderSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="_Close">
                                            <property name="minimumSize">
                                                <size>
                                                    <width>26</width>
                                                    <height>26</height>
                                                </size>
                                            </property>
                                            <property name="maximumSize">
                                                <size>
                                                    <width>26</width>
                                                    <height>26</height>
                                                </size>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                            <property name="iconSize">
                                                <size>
                                                    <width>16</width>
                                                    <height>16</height>
                                                </size>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QWidget" name="_Body" native="true">
                                <layout class="QVBoxLayout" name="_BodyLayout">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>12</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>12</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QWidget" name="_QrCodeWrapper" native="true">
                                            <layout class="QVBoxLayout" name="_QrCodeWrapperLayout">
                                                <property name="spacing">
                                                    <number>12</number>
                                                </property>
                                                <property name="leftMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="topMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="rightMargin">
                                                    <number>0</number>
                                                </property>
                                                <property name="bottomMargin">
                                                    <number>0</number>
                                                </property>
                                                <item>
                                                    <widget class="QLabel" name="_QrCodeTitle">
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>0</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>16777215</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true"/>
                                                        </property>
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                        <property name="alignment">
                                                            <set>Qt::AlignCenter</set>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QWidget" name="_QrCodePanel" native="true">
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>136</width>
                                                                <height>136</height>
                                                            </size>
                                                        </property>
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>16777215</width>
                                                                <height>136</height>
                                                            </size>
                                                        </property>
                                                        <layout class="QHBoxLayout" name="_QrCodePanelLayout">
                                                            <property name="spacing">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="leftMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="topMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="rightMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <property name="bottomMargin">
                                                                <number>0</number>
                                                            </property>
                                                            <item>
                                                                <widget class="QLabel" name="_QrCode">
                                                                    <property name="sizePolicy">
                                                                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                                            <horstretch>0</horstretch>
                                                                            <verstretch>0</verstretch>
                                                                        </sizepolicy>
                                                                    </property>
                                                                    <property name="minimumSize">
                                                                        <size>
                                                                            <width>136</width>
                                                                            <height>136</height>
                                                                        </size>
                                                                    </property>
                                                                    <property name="maximumSize">
                                                                        <size>
                                                                            <width>136</width>
                                                                            <height>136</height>
                                                                        </size>
                                                                    </property>
                                                                    <property name="cursor">
                                                                        <cursorShape>PointingHandCursor</cursorShape>
                                                                    </property>
                                                                    <property name="styleSheet">
                                                                        <string notr="true"/>
                                                                    </property>
                                                                    <property name="text">
                                                                        <string/>
                                                                    </property>
                                                                </widget>
                                                            </item>
                                                        </layout>
                                                    </widget>
                                                </item>
                                                <item>
                                                    <widget class="QLabel" name="_QrCodeStatus">
                                                        <property name="sizePolicy">
                                                            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                                                                <horstretch>0</horstretch>
                                                                <verstretch>0</verstretch>
                                                            </sizepolicy>
                                                        </property>
                                                        <property name="minimumSize">
                                                            <size>
                                                                <width>0</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="maximumSize">
                                                            <size>
                                                                <width>16777215</width>
                                                                <height>26</height>
                                                            </size>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true"/>
                                                        </property>
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                        <property name="alignment">
                                                            <set>Qt::AlignHCenter|Qt::AlignTop</set>
                                                        </property>
                                                        <property name="wordWrap">
                                                            <bool>true</bool>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                        <item>
                            <widget class="QWidget" name="_Footer" native="true">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>50</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>50</height>
                                    </size>
                                </property>
                                <property name="styleSheet">
                                    <string notr="true"/>
                                </property>
                                <layout class="QHBoxLayout" name="_FooterLayout">
                                    <property name="spacing">
                                        <number>12</number>
                                    </property>
                                    <property name="leftMargin">
                                        <number>12</number>
                                    </property>
                                    <property name="topMargin">
                                        <number>0</number>
                                    </property>
                                    <property name="rightMargin">
                                        <number>12</number>
                                    </property>
                                    <property name="bottomMargin">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <spacer name="_FooterSpacer">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                    <item>
                                        <widget class="QLabel" name="_Register">
                                            <property name="cursor">
                                                <cursorShape>PointingHandCursor</cursorShape>
                                            </property>
                                            <property name="styleSheet">
                                                <string notr="true"/>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <spacer name="_FooterSpaces">
                                            <property name="orientation">
                                                <enum>Qt::Horizontal</enum>
                                            </property>
                                            <property name="sizeHint" stdset="0">
                                                <size>
                                                    <width>40</width>
                                                    <height>20</height>
                                                </size>
                                            </property>
                                        </spacer>
                                    </item>
                                </layout>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
